client/src/main/java/org/spine3/protobuf/TypeToClassMap.java

```diff
@@ -74,7 +74,12 @@
      *
      * <p>For example, for a key {@code spine.base.EventId}, there will be the value {@code org.spine3.base.EventId}.
      */
-    private static final BiMap<TypeName, ClassName> NAMES_MAP = Builder.build();
+    private static final BiMap<TypeName, ClassName> TYPE_TO_CLASS_MAP = Builder.build();
```

Comment:
I think we should use `lowerCamelCase` for this static field. Have a look at [Google Style Guide on constants](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) and [non-constants](https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names). This member doesn't _feel_ like a constant event that it's immutable. Why?

Suppose we want to have a map of types that is updated as new types are dynamically added to the application. We don't do it now, but it might be an option in the future. Or, we'd want to use this class on design time, and add types dynamically as the user works in IDE.  Would the map still be in upper case then?